content/v0.80.0.md

```diff
@@ -0,0 +1,478 @@
++++
+title = "0.80.0 Release Notes"
+date = 2022-12-21
++++
+
+##  1. <a name='Tableofcontents'></a>Table of contents
+
+<!-- vscode-markdown-toc -->
+* 1. [Table of contents](#Tableofcontents)
+* 2. [Summary](#Summary)
+* 3. [How to upgrade](#Howtoupgrade)
+* 4. [New features](#Newfeatures)
+	* 4.1. [ReelMagic support](#ReelMagicsupport)
+	* 4.2. [Mouse improvements](#Mouseimprovements)
+	* 4.3. [Ability to use local configs via Finder on macOS](#AbilitytouselocalconfigsviaFinderonmacOS)
+* 5. [General improvements](#Generalimprovements)
+	* 5.1. [`more.com` command](#more.comcommand)
+	* 5.2. [On-the-fly changes](#On-the-flychanges)
+	* 5.3. [Post-rendered screenshots](#Post-renderedscreenshots)
+	* 5.4. [More bundled OpenGL shaders](#MorebundledOpenGLshaders)
+* 6. [Bug and regression fixes](#Bugandregressionfixes)
+* 7. [Translation improvements](#Translationimprovements)
+* 8. [Documentation updates](#Documentationupdates)
+* 9. [Contributors](#Contributors)
+* 10. [Thank you](#Thankyou)
+
+<!-- vscode-markdown-toc-config
+	numbering=true
+	autoSave=true
+	/vscode-markdown-toc-config -->
+<!-- /vscode-markdown-toc -->
+
+
+##  2. <a name='Summary'></a>Summary
+
+ReelMagic DOS games are now playable thanks to the efforts of Jon Dennis, Chris
+Guthrie, and Joseph Whittaker, with testing and feedback from Vogons users Uka,
+DonutKing, and Vetz. Mouse emulation has been improved and now supports up to
+seven independent mice, enabling split-screen play in games like Settlers 1 and
+Settlers II. More settings, including video, audio, and MIXER effects, can now
+be changed on the fly. A new `--working-dir` command line flag allows DOSBox
+Staging to launch from a specific working directory. Repo packagers should note
+that `SDL2-image` is now a dependency.
+ 
+Read on for all the details!
+
+##  3. <a name='Howtoupgrade'></a>How to upgrade
+
+Get started by downloading the latest version and updating your configuration
+file:
+
+**Downloads:**
+
+* [Linux](/downloads/linux/)
+* [Windows](/downloads/windows/)
+* [macOS](/downloads/macos/)
+
+**Upgrade your configuration file:**
```

Comment:
Please use the next heading, `###` in this case, and remove the `:` from the end.